en/cpp/quickstart.md

```diff
@@ -1,72 +1,52 @@
 # QuickStart
 
-This quickstart gets you started developing MAVSDK C++ core applications on macOS and Ubuntu Linux.
-It demonstrates how to run a simple MAVSDK example against a simulated vehicle, and observe the operation using the console and in a ground station.
+MAVSDK is essentially a C++ library (or rather multiple library files) that you can include and link in your C++ application.
 
 
-## Installation
+## Installation {#installation}
+
+First install the MAVSDK library on your platform.
+
+The latest release can be installed from a prebuilt binary. If you need the latest state or want to debug the library itself, you need to [build it from source](../contributing/build.md).
```

Comment:
I wonder if we should omit this line. After all, it's a quickstart.

Reply:
Exactly, thanks.